testkit/result.go

```diff
@@ -130,3 +130,42 @@ func (res *Result) CheckAt(cols []int, expected [][]interface{}) {
 	need := fmt.Sprintf("%s", expected)
 	res.require.Equal(need, got, res.comment)
 }
+
+// CheckContains checks whether the result contains the expected string
+func (res *Result) CheckContain(expected string) {
+	for _, row := range res.rows {
+		for _, colValue := range row {
+			if strings.Contains(colValue, expected) {
+				return
+			}
+		}
+	}
+	comment := fmt.Sprintf("the result doesn't contain the exepected %s", expected)
+	res.require.Equal(true, false, comment)
+}
+
+// CheckContains checks whether the result contains the expected string
```

Comment:
From the CI:
  ✘  https://revive.run/r#exported  comment on exported method Result.CheckContainMore should be of the form "CheckContainMore ..."  

  testkit/result.go:147:1